wildfly-standalone-centos/README.md

```diff
@@ -1,12 +1,12 @@
-# VM-Redhat - WildFly 16.0.0.Final standalone mode
+# Java on Azure | Red Hat WildFly 18.0.1.Final on CentOS 8 (standalone mode)
 <a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FSpektraSystems%2Fredhat-mw-cloud-quickstart%2Fmaster%2Fwildfly-standalone-centos%2Fazuredeploy.json" target="_blank">
     <img src="http://azuredeploy.net/deploybutton.png"/>
 </a>
 <a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FMicrosoft%2Fredhat-mw-cloud-quickstart%2Fmaster%2Fwildfly-standalone-centos%2Fazuredeploy.json" target="_blank">
     <img src="http://armviz.io/visualizebutton.png"/>
 </a>
 
-`Tags: WILDFLY, Red Hat, CentOS`
+`Tags: WildFly, Red Hat, CentOS, Java, Java EE, Jakarta EE, Azure, Azure VM`
```

Comment:
I have added the Tags as : "WildFly 18.0.1.Final, Red Hat, CentOS, Java, Java EE, Jakarta EE, Azure, Azure VM". Please suggest whether we can keep the version of WildFly here.